### DISCUSSION TITLE
Restrict the ability to close discussion to maintainers

### DISCUSSION THREAD

**Discussion by markseuffert** [community/community#52698] [Discussions]
### Select Topic Area

Product Feedback

### Body

GitHub Discussions now supports the ability to [close discussions](https://github.blog/changelog/2023-03-14-adding-the-ability-to-close-github-discussions/).

At the moment users can close discussions. This causes problems for a community that uses discussions mainly for Q&A. First there are buttons to `Close discussion` and `Mark as answer`, users may close a discussion instead of selecting an answer. Second closed questions are not shown per default, users may no longer find them and ask the same question again. At the moment most actions that [manage discussions](https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-discussions) are restricted to maintainers, for example to repository administrator or people with triage or greater access to a repository.

It would be great if you could restrict this new feature to maintainers. Let me know if you need more information.

**Comment by jamesgeorgewilliams** [comment]
This feature would be very beneficial for using discussions as a Q&A.  The following is my message on the "close discussions" thread.
https://github.com/orgs/community/discussions/3097#discussioncomment-5392051

---

It would be nice if you are able to disable this new feature, or limit closing to only maintainers/contributors of a repository.

If using discussions as a Q&A, `is:open` is the default filter. 

That means if a user searches discussions for their problem, and is unfamiliar with the UI, they might not find the solution as a `closed` discussion is not visible by default. This can lead to them creating a new discussion even though it has been answered already.

Limiting closing to maintainers/contributors, or a default filter of `All` discussions would be preferable for some cases.

**Comment by armandino** [comment]
Another vote for restricting the ability to close a discussion to maintainers.

Giving users the option to close Q&A discussions makes it harder to find the information later. It also creates additional work for maintainers having to re-open closed discussions.

**Comment by Daeraxa** [comment]
Alternatively I think I'd like an option for the default filter to not default to open only. I'm quite a fan of people opening up a new topic linking to the original rather than resurrecting it so for this the discovery of the topic is more important than the closing of the topic.

**Comment by markseuffert** [comment]
Removing that default filter is also mentioned in #64980.
